src/top/data/transforms.py

```diff
@@ -98,7 +100,8 @@ def _compute_gaussian_kernel(opts: Settings, device: th.device):
             opts.kernel_size,
             device=device) - opts.kernel_size // 2
         delta = th.exp_(-th.square_(delta) / (2.0 * opts.sigma**2))
-        delta /= delta.sum()
+        # delta /= delta.sum()
```

Comment:
This is a side-effect, could be removed prior to merging. I'm testing whether it makes more sense to set the gaussian kernel sum to 1 or max to 1.